### ISSUE TITLE
undefined method 'set_allowed?'

### ISSUE THREAD

**Issue by curthasselschwert** [kristianmandrup/acts_as_list_mongoid#3]
First I want to thanks for porting this gem. Whenever I try to us this on a document in my app, I keep receiving the error <code>undefined method 'set_allowed?'</code>. I've forked your repo and run the tests and am getting the same error. I've tried to trace the stack, reviewing the code in both of your other gems, mongoid_embedded_helper and mongoid_adjust, but I can't seem to figure out where the <code>'set_allowed?'</code> method is being called. I'd be happy to try and create a patch if you could point me in the right direction. My app is using <code>rails 3.1.0.rc4</code> with <code>mongoid 2.0.2</code>. Thanks!

**Comment by kristianmandrup** [comment]
Chek out this closed issue (by @saberma)

https://github.com/kristianmandrup/acts_as_list_mongoid/issues/1
1. 'set_allowed?' was deleted?

Maybe you can ask @saberma about this. 
Would also be cool if you could implement the feature requested by @KieranP. To add a class level macro acts_as_list instead of including the module.I will look into the #set_allowed? problem now. Thanks!

**Comment by kristianmandrup** [comment]
Looks like the set_allowed? was previously removed. Check old commits by @saberma perhaps? 
I just pushed some fixes and merged with @astevens. Enjoy!

**Comment by kristianmandrup** [comment]
Is finally fixed :) I just released 0.3.2 of mongoid_embedded_helper and also added #act_as_list macro

**Comment by curthasselschwert** [comment]
Great news! I went ahead and wrote a gem for lists of embedded documents specifically. Since they are stored as an array, I implemented functionality without the need for a <code>position</code> column in the document. Take a look at https://github.com/trait/mongoid_embedded_list. It's my first attempt at a gem, and unfortunately isn't hosted on rubygems.org yet because the site seems to be having problems the past few days. If any of the functionality looks useful to  you, maybe we can combine our efforts?
